simulation/rpg_rotors_interface/launch/basics/base_quad_simulation.launch

```diff
@@ -22,7 +23,7 @@
 
   <!-- Gazebo stuff to spawn the world !-->
   <env name="GAZEBO_MODEL_PATH" 
-      value="${GAZEBO_MODEL_PATH}:$(find rotors_gazebo)/models"/>
+      value="${GAZEBO_MODEL_PATH}:$(find rotors_gazebo)/models:$(arg custom_models)"/>
```

Comment:
Don't you want to keep the original line that ads "$(find rotors_gazebo)/models" to the path and then also add your custom directory to the path? Is this even required, aka doesn't this search recursively in the "models" directory?

Reply:
I agree that we should add both, Please check if it is required.

Reply:
Gazebo searches recursively in the default directory ~/.gazebo/models and in the above listed directories for simulation models. To avoid copying models from a repo to either the default directory or "rotors_gazebo/models", an additional directory could be passed.

Adding the additional directory to the env variable in the calling launch file did not work for me, so I opted for this solution.

Reply:
I think I misunderstood something in the beginning but now it makes sense and looks good to me